examples/sigstore-go-signing/main.go

```diff
@@ -180,7 +188,7 @@ func main() {
 		}
 		for _, tsaURL := range tsaURLs {
 			tsaOpts := &sign.TimestampAuthorityOptions{
-				URL:     tsaURL,
+				URL:     fmt.Sprintf("%s/api/v1/timestamp", tsaURL),
```

Comment:
maybe this would make more sense as an implementation detail of TimestampAuthority itself but the API docstring seemed clear: it's the api endpoint, not base url